components/var_dumper.rst

```diff
@@ -117,22 +120,71 @@ server, which outputs it to its own console or to an HTML file:
 
 Inside a Symfony application, the output of the dump server is configured with
 the :ref:`dump_destination option <configuration-debug-dump_destination>` of the
-``debug`` package.
+``debug`` package:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/debug.yaml
+        debug:
+           dump_destination: "tcp://%env(VAR_DUMPER_SERVER)%"
+
+    .. code-block:: xml
+
+        <!-- config/packages/debug.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/debug"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:debug="http://symfony.com/schema/dic/debug"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/debug http://symfony.com/schema/dic/debug/debug-1.0.xsd">
+
+            <debug:config dump-destination="tcp://%env(VAR_DUMPER_SERVER)%" />
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/debug.php
+        $container->loadFromExtension('debug', array(
+           'dump_destination' => 'tcp://%env(VAR_DUMPER_SERVER)%',
+        ));
 
-Outside a Symfony application, use the ``ServerDumper`` class::
+Outside a Symfony application, use the :class:``Symfony\\Component\\VarDumper\\Dumper\\ServerDumper`` class::
```

Comment:
The `:class:` directive only requires 1 backtick instead of the 2 used in `<code>`. Thanks!